src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.styles.ts

```diff
@@ -0,0 +1,58 @@
+import palette from '~/styles/app-theme/app.pallete'
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  container: {
+    maxWidth: '400px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '16px',
+    p: '20px',
+    mt: '24px',
+    borderRadius: '2px 2px 6px 6px',
+    borderTop: `4px solid ${palette.basic.turquoise}`,
+    background: `${palette.basic.white}`,
+    '&:hover': {
+      boxShadow: '0px 3px 16px 2px rgba(144, 164, 174, 0.56)'
```

Comment:
import 'commonHoverShadow' and paste it as a value for boxShadow property because it has same value

Reply:
done